nixos/modules/system/boot/plymouth.nix

```diff
@@ -225,6 +238,11 @@ in
         plymouth-switch-root-initramfs.wantedBy = [ "halt.target" "kexec.target" "plymouth-switch-root-initramfs.service" "poweroff.target" "reboot.target" ];
         plymouth-switch-root.wantedBy = [ "initrd-switch-root.target" ];
       };
+      # Link in runtime files before starting
+      services.plymouth-start.preStart = ''
+        mkdir -p /run/plymouth
+        ln -sf /etc/plymouth/{plymouthd.defaults,themes,plugins} /run/plymouth/
+      '';
```

Comment:
We can't use tmpfiles here?

Reply:
@SuperSandro2000 `tmpfiles` runs after `local-fs.target`, which is *far* too late for plymouth.